src/docs/product/sentry-basics/sampling/index.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Sampling
+sidebar_order: 25
+redirect_from:
+  - /product/sentry-basics/metrics/
+description: "Learn more about the different ways that you can sample the transaction events that you send to Sentry."
+---
+
+<Note>
+
+This feature is in beta and is only available if your organization is participating in its limited release. Features in beta are still in-progress and may have bugs. We recognize the irony. If you’re interested in participating, [join the waitlist](https://sentry.io/for/performance/#updates-signup).
+
+</Note>
+
+When you're using [sentry.io](https://sentry.io), the transaction information that you see is generated by one of two types of events:
+
+- Processed events - All the transaction events that you send to Sentry, based on your client-side sample rate in your SDK.
+- Indexed events - A sub-set of your processed transaction events, based on your [server-side sample rate](/product/data-management-settings/server-side-sampling/). Underlies _events-based_ data.
+
+We call this ability to set sample these in these different ways _dynamic sampling_. Dynamic sampling is a suite of features that allow you to view more accurate performance metrics based on all the transaction events you send to Sentry (or your _processed events_), while retaining a smaller number of more relevant transaction events that you can explore in full detail (your _indexed events_).
+
+![Image of a breakdown of event types](metrics-events-subsets.png)
+
+## Client-Side (SDK) Sampling
+
+_Processed_ transaction events can include all the events that occur in your application or a sub-set of those. The number of these transactions that you send to Sentry is based on your client-side sample rate or the sample rate that you set in the SDK. These transactions represent aggregate data. This is used in areas of the application like **Performance** and **Dashboards**. In [sentry.io](https://sentry.io), you may see the data generated from these transaction events referred to as _metrics_ or _metric-backed_.
+
+Because these transactions represent aggregate data, Sentry doesn't store the full transaction data. So when you're viewing information in [sentry.io](https://sentry.io) that's based on these events, you can't drill down into the specifics of them.
```

Comment:
```suggestion
Because these transactions represent aggregate data, Sentry doesn't store the full transaction data. So when you're viewing information in [sentry.io](https://sentry.io) that's based on these events, you can't drill down into the specifics of individual events.
```